### ISSUE TITLE
Error: Severity for rule 'object-literal-key-quotes not found

### ISSUE THREAD

**Issue by adambowles** [palantir/tslint#2596]
### Bug Report

- __TSLint version__: 5.1.0
- __TypeScript version__:
- __Running TSLint via__: CLI (Node.js 7.4.0)

with `tslint.json` configuration:

```json
{
    "rules": {
        "object-literal-key-quotes": {
            "options": [ "as-needed" ],
            "severity": "warn"
        }
    }
}

```

#### Actual behavior

```
➜ test ./node_modules/.bin/tslint -c tslint.json ./**/*
No valid rules have been specified
/home/adambowles/Projects/test/node_modules/tslint/lib/runner.js:116
            throw error;
            ^

Error: Severity for rule 'object-literal-key-quotes not found
    at Linter.lint (/home/adambowles/Projects/test/node_modules/tslint/lib/linter.js:109:23)
    at Runner.processFiles (/home/adambowles/Projects/test/node_modules/tslint/lib/runner.js:157:20)
    at Runner.run (/home/adambowles/Projects/test/node_modules/tslint/lib/runner.js:108:18)
    at Object.<anonymous> (/home/adambowles/Projects/test/node_modules/tslint/lib/tslint-cli.js:140:6)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
```

#### Expected behavior
Should throw a warning for violations (if any)

**Comment by adambowles** [comment]
Doesn't occur if I run `./node_modules/.bin/tslint -c tslint.json ./**/*.ts` or `./node_modules/.bin/tslint -c tslint.json ./**/*.tsx`

**Comment by olore** [comment]
I am able to reproduce this in the tslint project itself by running `./node_modules/.bin/tslint -c tslint.json ./**/*`, there is no problem if I append `.ts` to the wildcard, as you indicated.

I don't think tslint is meant to be run against non-TypeScript files globbed by `**/*`

If you are trying to lint both `ts ` and `tsx` files you can use this glob: `**/*.ts?`

**Comment by nchen63** [comment]
likely fixed by https://github.com/palantir/tslint/pull/2516
